### DISCUSSION TITLE
Welcome to OSSM-hardware Discussions!

### DISCUSSION THREAD

**Discussion by JC-OpenHW** [KinkyMakers/OSSM-hardware#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jakob4play** [comment]
Hello all, have a HISMITH tabletop  sex machine which got me interested along with some friends..does anyone sell these after putting them together? Or if your ever in need of a tester I know someone!!

**Comment by Kronologic81** [comment]
I've created a subversion of the double double adapter with a retention pad to avoid tear the rubber under preasure, because the adapter acts as an spike.

If you have doubts feel free to coment it to me...

[double double to the moon 24mm v1 with base.zip](https://github.com/KinkyMakers/OSSM-hardware/files/11871860/double.double.to.the.moon.24mm.v1.with.base.zip)
